management/ia/rules_test.go

```diff
@@ -32,7 +32,16 @@ func TestRulesAPI(t *testing.T) {
 
 	t.Run("add", func(t *testing.T) {
 		t.Run("normal", func(t *testing.T) {
-			params := createAlertRuleParams(templateName, channelID)
+			params := createAlertRuleParams(templateName, channelID, true)
```

Comment:
can we replace these `true/false` values with some constants to make it explicit?

Reply:
I replaced the bool with a pointer to a fiter so that we can check for nil

Reply:
> Can we check if channels and filters are actually added?

we can check for it, but we would have to call List for that iterate over it to check, do you want to add another API call and a for loop to all testcase, because we already check for it on managed in our unit tests here https://github.com/percona/pmm-managed/pull/609/files#diff-7df81ba599594e0fb11c381bb999cc71b71361ba3a9d852784f04e67c37b223cR99 ?